fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/LambdaFunctionOperator.java

```diff
@@ -0,0 +1,100 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.optimizer.operator.scalar;
+
+import com.google.common.collect.Lists;
+import com.starrocks.catalog.Type;
+import com.starrocks.sql.optimizer.base.ColumnRefSet;
+import com.starrocks.sql.optimizer.operator.OperatorType;
+
+import java.util.List;
+import java.util.Objects;
+
+public class LambdaFunctionOperator extends ScalarOperator {
```

Comment:
Should write some test cases for it?

Reply:
added some tests for expression analyzer.